stack/rest/src/main/java/org/apache/usergrid/rest/organizations/OrganizationResource.java

```diff
@@ -143,21 +143,21 @@ public ApplicationResource getApplicationByName( @PathParam("applicationName") S
     }
 
 
-    @Path("applications/{applicationName}")
+    @Path("applications/{applicationName: [^<>/]+}")
```

Comment:
If you want to understand why the output always includes the unescaped URI, take a look at [OrganizationApplicationNotFoundException.java](https://github.com/apache/incubator-usergrid/blob/master/stack/rest/src/main/java/org/apache/usergrid/rest/exceptions/OrganizationApplicationNotFoundException.java#L38)